### PR TITLE
test(extension): add tests for LW-5104

### PR DESCRIPTION
**Pull request by wklos-iohk**
# Checklist

- [X] JIRA - LW-10618
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

### CODE REVIEW

**Review by emiride — APPROVED** [review]
Looks good to me

File: packages/e2e-tests/src/elements/TopUpWalletCard.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  get buyAdaButton(): ChainablePromiseElement<WebdriverIO.Element> {
+    return $(this.BUY_ADA_BUTTON);
+  }
+
+  get disclaimer(): ChainablePromiseElement<WebdriverIO.Element> {
+    return $(this.DISCLAIMER);
+  }
+
+  async clickByAdaButton(): Promise<void> {
```
Comment (ljagiela) [inline packages/e2e-tests/src/elements/TopUpWalletCard.ts:36]: should it be clickBuy... ?

File: packages/e2e-tests/src/elements/TopUpWalletSmallCard.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  get buyAdaButton(): ChainablePromiseElement<WebdriverIO.Element> {
+    return $(this.BUY_ADA_BUTTON);
+  }
+
+  get disclaimer(): ChainablePromiseElement<WebdriverIO.Element> {
+    return $(this.DISCLAIMER);
+  }
+
+  async clickByAdaButton(): Promise<void> {
```
Comment (ljagiela) [inline packages/e2e-tests/src/elements/TopUpWalletSmallCard.ts:26]: same here Buy ?

File: packages/e2e-tests/src/features/FiatOnRampOffRampBanxaExtended.feature [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    Then Banxa's transaction page is opened in a new tab
+
+  @LW-10598
+  Scenario: Fiat On & Off Ramp - Banxa - widget - screen width <= 1280px
+    Given I switch network to: "Mainnet" in extended mode
+    When I navigate to Tokens extended page
+    And I resize the window to a width of: 1281 and a height of: 1080
+    Then Banxa widget is displayed
+    When I resize the window to a width of: 1280 and a height of: 1080
+    Then Banxa widget is not displayed
```
Comment (ljagiela) [inline packages/e2e-tests/src/features/FiatOnRampOffRampBanxaExtended.feature:50]: Cosmetic: Sometimes you are using Banxa and sometimes Banxa's.

**Review by ljagiela — CHANGES_REQUESTED** [review]

**Review by ljagiela — APPROVED** [review]

**Pull request merged**